shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/StopDetailsDepartures.kt

```diff
@@ -2,9 +2,9 @@ package com.mbta.tid.mbta_app.model
 
 import com.mbta.tid.mbta_app.model.response.AlertsStreamDataResponse
```

Comment:
question (non-blocking): Do you see `StopDetailsDepartures` as being fully replaced by `List<StopsAssociated>` in the future? I think removing this data type in the future could help to further the consistency between nearby transit & stop details, but certainly not needed as part of this refactor!